.gitignore

```diff
@@ -78,4 +78,4 @@ prime-router/result_files/*
 prime-router/routed_files/*
 
 # Hashicorp Vault Secrets
-prime-router/.vault/env/**
\ No newline at end of file
+prime-router/.vault/env/**
```

Comment:
There is an extra line that got added here. Don't mean to be a stickler, but our PRs should only contains changes to files we actively developed. If we're making code formatting changes, that should be a separate PR. If we're having issues where different developer's IDEs are formatting code differently, we should call that out, and improve our linting and formatting rules so it's consistent across the team. This could be the source of some of our merge issues (different linting rules per dev).